use_cases/eluc/prescriptors/nsga2/train_prescriptors.py

```diff
@@ -1,32 +1,46 @@
+"""
+Script used to train NSGA-II prescriptors.
+Requires a config file with the same fields as shown in the
+test.json file in prescriptors/nsga2/configs
+"""
+
+import argparse
+import json
 from pathlib import Path
 
-from data import constants
 from data.eluc_data import ELUCData
 from prescriptors.nsga2.torch_prescriptor import TorchPrescriptor
 from predictors.neural_network.neural_net_predictor import NeuralNetPredictor
 
 if __name__ == "__main__":
+
+    # Load config
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config_path", type=str, required=True)
+    args = parser.parse_args()
+    with open(Path(args.config_path), "r", encoding="utf-8") as f:
+        config = json.load(f)
```

Comment:
The only argument we need is a config file. More customizability can come in the subset of data we use, what type of predictor we use, etc. but for now this simple one works